packages/debug/src/node/debug-model.ts

```diff
@@ -22,12 +22,12 @@
 // Some entities copied and modified from https://github.com/Microsoft/vscode/blob/master/src/vs/vscode.d.ts
```

Comment:
This file should have been kept in the common folder. The move has blocked browser-based adapters from working